project/plugins.sbt

```diff
@@ -22,7 +22,7 @@ addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "2.2.0")
 
```

Comment:
This PR change is inspired from this https://github.com/delta-io/delta-sharing/pull/763/files